modules/bitgo/example/js/eth/list-wallet-addresses.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * List deployed receive addresses of a eth wallet and their given token balances.
+ *
+ * This tool will help you see how to use the BitGo API to easily list your
+ * BitGo wallets.
+ *
+ * Copyright 2022, BitGo, Inc.  All Rights Reserved.
+ */
+import { BitGo } from 'bitgo';
+const bitgo = new BitGo({ env: 'test' });
+
+const coin = 'teth';
```

Comment:
the token that is used in this example is related to gteth, and anyways we have depreciated teth. So it would be better to create new example with gteth

Reply:
Done.